WordPress/gutenberg
# Dimensions Tool: Change the conditions underwhich we display the scale control

## What?
Don't show the scale control when aspect ratio is set to "Original".

## Why?
[This PR](https://github.com/WordPress/gutenberg/pull/53274) changed the way we save the aspect ratio control which means that we save the aspect ratio even when it's set to the natural/original ratio. We shouldn't display the scale control when the aspect ratio is set to original, as it doesn't do anything.



## How?
Save the aspect ratio as `auto` when the selected ratio is the same as the natural ratio.

## Testing Instructions
1. Add an image to a post
2. Change the width/height using the drag handles
3. Confirm that the scale control doesn't display in the sidebar.

## Screenshots or screencast 
<img width="304" alt="Screenshot 2023-08-04 at 10 54 23" src="https://github.com/WordPress/gutenberg/assets/275961/183abd72-f645-4a23-a8ec-0d587860fbe0">

## Note
I also discovered a bug whereby resizing image that had an aspect ratio set caused the block to crash. This is because the block was saving the aspect ratio as a number instead of a string. I have reverted the aspect ratio we save back to a string.
